assets/js/field-config.js

```diff
@@ -707,20 +707,19 @@
 				result = 0;
 			}
 
-			if ( null === lastValue || result !== lastValue ) {
```

Comment:
This condition was removed and is the only change in the code here.

Reply:
I get why this would cause the bug. I THINK that my intent here was to have this do two things:

- Not run a state mutation when the form is first loaded when lastValue would be null. Unfrtounetly, it could be null beacuse of the conditonal hide. Hence this bug.
- Not run a state mutation for no reason -- `result !== lastValue` . If result and lastValue are equal, then it's not meaningful to update lastValue or mutateState.

Maybe the real issue is that when `lastValue` is null, that could mean one of two things. It could mean field has never been run through this system before or that field is being undhidden. Can we represent one of those states with `undefined` instead? `0` and `false` could be valid field values, so I do not want to use those.